sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/CheckAnalysis.scala

```diff
@@ -859,7 +866,7 @@ trait CheckAnalysis extends PredicateHelper with LookupCatalog with QueryErrorsB
               summary = j.origin.context.summary)
 
           // TODO: although map type is not orderable, technically map type should be able to be
-          // used in equality comparison, remove this type check once we support it.
+          //   used in equality comparison, remove this type check once we support it.
```

Comment:
unnecessary changes:
```suggestion
          // used in equality comparison, remove this type check once we support it.
```